launch_testing/launch_testing/test_runner.py

```diff
@@ -223,9 +222,9 @@ def run(self):
         # We will return the results as a {test_run: TestResult)}
         results = {}
 
-        for index, run in enumerate(self._test_runs):
+        for run in self._test_runs:
             if len(self._test_runs) > 1:
-                print('Starting test run {}'.format(index + 1))
+                print('\n***** Starting test run {} *****'.format(run))
```

Comment:
Please don't use "random" starts to make a certain print statement stand out like this. If multiple parts in the code do that the output becomes even more cluttered and difficult to read. I just took the liberty to apply that change to get this merged quickly: 6071175be995f11d6ee6b8e3017402877418b4cf

Reply:
@dirk-thomas Can we at least keep the extra line of whitespace please?

Reply:
Sure: b974f6498ae2da1fc4b95d2fc105159f66a69354

Reply:
Thanks.

There's probably a future issue to clean up the output from launch_testing to make it easier to read.  One of the beefs I had with rostest was it was often difficult to visually parse what was going on.  

Right now the output from unittest gets intermingled with the output from launch so it's hard to tell what's a test result and what's a message about a process starting or stopping.  Also we don't intercept stdout in the passing case like pytest or nosetest does.  It's probably lower priority, but I'll open an issue and self-assign it